docs/Reference/Cakeshop/Overview.md

```diff
@@ -0,0 +1,79 @@
+# Getting started with Cakeshop
+
+## Download
+
+Binary packages are available on the [GitHub releases page](https://github.com/ConsenSys/cakeshop/releases).
+
+## Configuration
+
+Cakeshop is a Spring Boot application, so you may place an `application.properties` file in the working directory to override any default configuration values. For more info, see the [configuration page](Configuration.md).
+
+## Running via GoQuorum Wizard
+
+The easiest way to use Cakeshop is to generate a GoQuorum network with [GoQuorum Wizard](../../HowTo/GetStarted/Wizard/GettingStarted.md) and choose to deploy Cakeshop alongside the network.
```

Comment:
```suggestion
The recommended way to use Cakeshop is to generate a GoQuorum network with [GoQuorum Wizard](../../HowTo/GetStarted/Wizard/GettingStarted.md) and choose to deploy Cakeshop alongside the network.
```
statements like easy, easiest, simple,... are to avoid